service-rpc/src/main/java/org/apache/iotdb/rpc/TSStatusCode.java

```diff
@@ -63,10 +63,18 @@
   READ_ONLY_SYSTEM_ERROR(502),
   DISK_SPACE_INSUFFICIENT_ERROR(503),
   START_UP_ERROR(504),
+
   WRONG_LOGIN_PASSWORD_ERROR(600),
   NOT_LOGIN_ERROR(601),
   NO_PERMISSION_ERROR(602),
   UNINITIALIZED_AUTH_ERROR(603),
+
+  PARTITION_NOT_READY(700),
+  TIME_OUT(701),
+  NO_LEADER(702),
+  UNSUPPORTED_OPERATION(703),
+  NODE_READ_ONLY(704),
+
```

Comment:
It will be better if you update the `docs/UserGuide/4-Client/7-Status Codes.md` file as well.

Reply:
The same as above: this will be added when the distributed version is fully ready.